internal/provisioner/fluentbit.go

```diff
@@ -80,17 +79,53 @@ func (f *fluentbit) NewHelmDeployment(logger log.FieldLogger) *helmDeployment {
 	if err != nil {
 		logger.WithError(err).Error("unable to lookup private zone name")
 	}
+
+	zoneID, err := f.awsClient.GetPrivateZoneID(logger)
+	if err != nil {
+		logger.WithError(err).Error("unable to find Zone ID")
+	}
+
+	tag, err := f.awsClient.GetTagByKeyAndZoneID(aws.DefaultAuditLogsCoreSecurityTagKey, zoneID, logger)
+	if err != nil {
+		logger.WithError(err).Error("failed to find Tag:AuditLogsCoreSecurity")
+	}
+	if tag == nil {
+		logger.Info("unable to find tag")
+		tag = &aws.Tag{}
+	}
+
```

Comment:
if `err` was not `nil`, then `tag` is `nil` at this point and line 97 will panic